apps/client/src/app/pages/home/header/Header.tsx

```diff
@@ -0,0 +1,47 @@
+import NotificationIcon from '@home-assets/notifications-icon.png';
+import UserIcon from '@home-assets/user-icon.png';
+import Button from '@map-shared-components/Button';
+
+function Header({ isLoggedIn, username }: HeaderProps) {
```

Comment:
If your username is optional like you put in your types, username should have a default value in this function